aca_lts_eval.py

```diff
@@ -176,10 +177,45 @@ def select_ri_guide_stars(ra, dec, cone_stars):
     return G_RI_CAT_CACHE[id_key]
 
 
-def t_lose_star(mag, ref_mag=10.3, ref_tccd=-11.5, scale_4c=None):
+def guide_count(mags, tccd=-11.5):
+    """
+    Given mags from guide stars and a temperature, calculate a guide star
+    count using signal-to-noise scaled mag thresholds.
+    """
+    thresh1 = snr_mag_for_tccd(tccd, ref_mag=10.0)
+    thresh2 = snr_mag_for_tccd(tccd, ref_mag=10.2)
+    thresh3 = snr_mag_for_tccd(tccd, ref_mag=10.3)
+    counts = np.zeros_like(mags)
+    counts[mags <= thresh1] = 1.0
+    counts[(mags <= thresh2) & (mags > thresh1)] = 0.75
+    counts[(mags <= thresh3) & (mags > thresh2)] = 0.5
+    return np.sum(counts)
+
+
+def snr_mag_for_tccd(tccd, ref_mag=10.3, ref_tccd=-11.5, scale_4c=None):
```

Comment:
I suppose I should set ref_tccd to -10.2 . 10.3 @ -11.5 was our original reference, but if we are implementing the current star counts in starcheck and ORViewer at the 10.0, 10.2, and 10.3 mag limits with a planning limit of -10.2, this ref temperature should probably come up.

Reply:
Also, if we move this to chandra_aca does it belong in star_probs or transform?

Reply:
transform.